self_hosted/runs_wrong.txt

```diff
@@ -0,0 +1,165 @@
+tests/should_succeed/plusplus_minusminus.jou
+tests/should_succeed/pointer.jou
+tests/should_succeed/printf.jou
+tests/should_succeed/return_string.jou
+tests/should_succeed/return_void.jou
+tests/should_succeed/sizeof.jou
+tests/should_succeed/sscanf.jou
+tests/should_succeed/stderr.jou
+tests/should_succeed/string_syntax.jou
+tests/should_succeed/strlibtest.jou
+tests/should_succeed/undefined_value_warning.jou
+tests/should_succeed/unreachable_warning.jou
+tests/should_succeed/unused_import.jou
+tests/syntax_error/and_or_chaining.jou
+tests/syntax_error/array_size.jou
+tests/syntax_error/bad_addressof.jou
+tests/syntax_error/bad_array.jou
+tests/syntax_error/bad_byte.jou
+tests/syntax_error/bad_field.jou
+tests/syntax_error/chained_eq.jou
+tests/syntax_error/chained_le.jou
+tests/syntax_error/class_init_js_syntax.jou
+tests/syntax_error/dot_after_e.jou
+tests/syntax_error/double_assignment.jou
+tests/syntax_error/double_not.jou
+tests/syntax_error/double_with_letters_after.jou
+tests/syntax_error/ee.jou
+tests/syntax_error/import_after_def.jou
+tests/syntax_error/import_missing_comma.jou
+tests/syntax_error/import_missing_comma_with_parens.jou
+tests/syntax_error/import_missing_dot.jou
+tests/syntax_error/import_missing_quotes.jou
+tests/syntax_error/indexing.jou
+tests/syntax_error/infinite_c_style_for.jou
+tests/syntax_error/missing_field_names.jou
+tests/syntax_error/missing_import_keyword.jou
+tests/syntax_error/missing_number_after_eminus.jou
+tests/syntax_error/missing_second_equal_sign.jou
+tests/syntax_error/multidot_float.jou
+tests/syntax_error/python_style_for.jou
+tests/syntax_error/self_outside_class.jou
+tests/syntax_error/triple_equals.jou
+tests/too_long/name.jou
+tests/wrong_type/arg.jou
+tests/wrong_type/array_mixed_types.jou
+tests/wrong_type/array_mixed_types_ptr.jou
+tests/wrong_type/array_to_ptr.jou
+tests/wrong_type/array_vararg.jou
+tests/wrong_type/arrow_operator_not_class.jou
+tests/wrong_type/arrow_operator_not_pointer.jou
+tests/wrong_type/arrow_operator_not_pointer_method.jou
+tests/wrong_type/assign_to_deref_non_pointer.jou
+tests/wrong_type/assign_void.jou
+tests/wrong_type/assign_with_type.jou
+tests/wrong_type/bool_main.jou
+tests/wrong_type/brace_init_arg.jou
+tests/wrong_type/cannot_be_indexed.jou
+tests/wrong_type/class_member_assign.jou
+tests/wrong_type/class_member_init.jou
+tests/wrong_type/deref_non_pointer.jou
+tests/wrong_type/dot_operator.jou
+tests/wrong_type/elif.jou
+tests/wrong_type/enum_member_from_class.jou
+tests/wrong_type/enum_to_int.jou
+tests/wrong_type/float_and_double.jou
+tests/wrong_type/for.jou
+tests/wrong_type/if.jou
+tests/wrong_type/index.jou
+tests/wrong_type/inplace_add_doesnt_go_back.jou
+tests/wrong_type/int_to_enum.jou
+tests/wrong_type/mod.jou
+tests/wrong_type/neg.jou
+tests/wrong_type/not.jou
+tests/wrong_type/or.jou
+tests/wrong_type/plusplus.jou
+tests/wrong_type/pointer_assignment.jou
+tests/wrong_type/pointer_eq.jou
+tests/wrong_type/return_value.jou
```

Comment:
And why is  `return_value.jou` in this list?

Reply:
Because we don't have local variables yet, and it is attempting to return a local variable. I will change the test to return an integer constant instead.